eng/pipelines/common/global-build-job.yml

```diff
@@ -63,6 +63,13 @@ jobs:
       - ${{ if and(eq(parameters.osGroup, 'Linux'), eq(parameters.osSubGroup, '_musl')) }}:
         - name: _osParameter
           value: /p:RuntimeOS=linux-musl /p:OutputRid=linux-musl-${{ parameters.archType }}
+      - ${{ if and(eq(parameters.osGroup, 'tvOS'), eq(parameters.osSubGroup, '_sim')) }}:
```

Comment:
I think we should use `osGroup` for `iOS-sim` and `tvOS-sim` rather than using this `osSubGroup` for consistency.